packages/mobile/src/screens/chat-screen/ChatScreen.tsx

```diff
@@ -331,6 +418,20 @@ export const ChatScreen = () => {
       topbarRight={topBarRight}
     >
       <ScreenContent>
+        {/* Everything inside the portal displays on top of all other screen contents. */}
+        <Portal hostName='ChatReactionsPortal'>
+          {shouldShowPopup && popupChatIndex !== null ? (
+            <ReactionPopup
+              reactionInfo={reactionInfo.current}
```

Comment:
nit: could pass each of the ref's props in separately to avoid the generic sounding "reactionInfo" within the component

Reply:
In fact, the `hasTail` and `isAuthor` could be calculated here using `chatMessages` and `popupIndex`, and don't need to be part of the ref imo

Reply:
I think maybe the only thing that needs to be part of the ref is the `y` position of the message